src/json-schema-content-provider.ts

```diff
@@ -62,12 +62,32 @@ export async function getJsonSchemaContent(uri: string, schemaCache: JSONSchemaC
     .catch((error: XHRResponse) => {
       // content not changed, return cached
       if (error.status === 304) {
-        return schemaCache.getSchema(uri);
+        const content = schemaCache.getSchema(uri);
```

Comment:
```suggestion
        const content = await schemaCache.getSchema(uri);
```
I was able to get the code working with the above change on windows with WSL but for some reason the code lens won't show up properly